### PR TITLE
Playground buttons

### PR DESCRIPTION
**Pull request by pickjasmine**
* Adds buttons to playground to copy the shown code and clear the selected resource/policy
* Buttons now take in `tooltip` prop to add a hover tooltip to the button

### CODE REVIEW

File: test/components/playground/SelectedPolicy.spec.js [hunk 1]
```diff
@@ -15,44 +15,101 @@
...
+      render(
+        <SelectedPolicy
+          policy={policy}
+          setPolicy={setPolicy}
+          clearEvaluation={clearEvaluation}
+        />
+      );
+    });
+
+    it("should render the rego policy label", () => {
```
Comment (alexashley) [inline test/components/playground/SelectedPolicy.spec.js:78]:
```suggestion
    it("should render the Rego policy label", () => {
```
Definitely a nitpick, but I think Rego is always capitalized. Same thing for a few of the other test titles.

File: components/Button.js [hunk 2]
```diff
@@ -29,23 +30,44 @@ const Button = (props) => {
...
+    tooltip = null,
     ...otherProps
   } = props;
 
   const { theme } = useTheme();
 
+  let tooltipId = null;
+
+  if (tooltip) {
+    tooltipId = `${label.replace(" ", "")}-toolTip`;
```
Comment (alexashley) [inline components/Button.js:42]:
```suggestion
    tooltipId = `${tooltip.replace(" ", "")}-toolTip`;
```
Should this be the tooltip value?

File: components/Button.js [hunk 3]
```diff
@@ -29,23 +30,44 @@ const Button = (props) => {
...
-    </button>
+    <>
+      {tooltip && (
+        <ReactTooltip
+          id={tooltipId}
+          effect={"solid"}
+          place={"top"}
+          delayShow={200}
+        >
+          <p>{tooltip}</p>
```
Comment (alexashley) [inline components/Button.js:54]: At least from the use in this PR, it seems like the tooltip value is always the same as the `label` prop. What do you think about having a prop called `showTooltip` (or something along those lines) that renders the label value as tooltip? Or is there a case where those might be different?

Comment (pickjasmine) [reply components/Button.js:54]: My thinking was any other button that isn't an icon would want a different value than the label but like you said, there's not currently use case for it. I can default to the label for now and update this logic if we find a different use case later.

File: components/Button.js (hunk 2 above)
Comment (pickjasmine) [reply components/Button.js:42]: This value does not really matter as long as it is consistent between the `ReactTooltip` `id` prop and the `Button` `data-for` prop

**Review by alexashley — APPROVED** [review]
:shipit:

**Pull request merged**